Placenote.framework/_CodeSignature/CodeResources

```diff
@@ -10,11 +10,11 @@
 		</data>
 		<key>Headers/libPlacenote.h</key>
 		<data>
-		jPXkxCylBGdrwUtg5yfObmqNXpY=
```

Comment:
what is this file? auto generated ?

Reply:
Yup. autogenerated. Was not added on this commit. Was always there.